packages/Datadog.Unity/Runtime/WebGL/DatadogWebGLRum.cs

```diff
@@ -0,0 +1,221 @@
+            public string service { get; set; }
+            public string env { get; set; }
+            public string version { get; set; }
+            public string proxy { get; set; }
+            // TODO: allowedTracingUrls
+            public float traceSampleRate { get; set; }
+            public string traceContextInjection { get; set; }
+            public bool trackFrustrations { get; set; }
+            public bool trackViewsManually { get; set; } = true;
+            public bool trackResources { get; set; }
+            public bool trackLongTasks { get; set; }
+            public List<string> enableExperimentalFeatures { get; set; } = new List<string>();
+        }
+
+        public void Init(DatadogConfigurationOptions options)
+        {
+            var browserSdkConfig = new BrowerSdkConfig
+            {
+                applicationId = options.RumApplicationId,
+                clientToken = options.ClientToken,
+                site = DatadogWebGLHelpers.SiteStringForSite(options.Site),
+                sessionSampleRate = options.SessionSampleRate,
+                service = options.ServiceName,
+                env = options.Env,
+                // TODO: Version
+                version = "unknown",
+                traceSampleRate = options.TraceSampleRate,
+                trackFrustrations = true,
+                trackResources = true,
+                trackLongTasks = true,
+                proxy = options.CustomEndpoint,
+            };
+            var configurationJson = JsonConvert.SerializeObject(browserSdkConfig);
+            // TODO: Check the return value of this
+            DDRum_InitRum(configurationJson);
+        }
+
+        public void StartView(string key, string name = null, Dictionary<string, object> attributes = null)
+        {
+            var viewName = key;
+            if (name != null)
+            {
+                viewName = name;
+            }
+
+            var attributesJson = "{}";
+            if (attributes != null)
+            {
+                attributesJson = JsonConvert.SerializeObject(attributes);
+            }
+
+            DDRum_StartView(viewName, attributesJson);
+        }
+
+        public void StopView(string key, Dictionary<string, object> attributes = null)
+        {
+
+        }
+
+        public void AddAction(RumUserActionType type, string name, Dictionary<string, object> attributes = null)
+        {
+            if (name == null)
+            {
+                return;
+            }
+
+            var attributesJson = "{}";
+            if (attributes != null)
+            {
+                attributesJson = JsonConvert.SerializeObject(attributes);
+            }
+
+            DDRum_AddAction(
+                name,
+                attributesJson);
+        }
+
+        public void StartAction(RumUserActionType type, string name, Dictionary<string, object> attributes = null)
+        {
+        }
```

Comment:
should we put `TODO` here?

and same for the similar empty methods below

Reply:
I'll add a comment that they're not currently supported in the Browser SDK. I'd rather not leave a TODO because they're not an immediate concern